### DISCUSSION TITLE
disallow inline hashtags in link text?

### DISCUSSION THREAD

**Discussion by edrex** [srid/emanote#166] [Ideas]
I'm trying out emanote with my notes archive and I am very impressed with its focused feature set.

I notice my tags list is flooded with tags of the form #[\d+], coming from the titles of issue tracker links. Looking at the source I see the hashTagSpec follows CommonMark's allowing other inline elements in link titles, ie [*ital* **bold** #hash](foo). Still, in the case of hashtags my gut says they should be disallowed in the titles of other links (mostly because I want to be able to drop links from the browser without sanitizing the titles), also Obsidian doesn't parse them. WDYT?

**Comment by srid** [comment]
At first I thought you were talking about disabling inline hashtags *altogether*, but you are actually asking to disable them in the body of a link text. That makes sense. We can just disable it in link inlines. Opened https://github.com/srid/emanote/issues/167

You use `\#23` to escape them, but that's not ideal.

**Reply by edrex** [reply]
I noticed escaping the hash works, and that's fine for manually-titled external links, and would be ok as an additional step when copying links from the browser. It would be nice if there was a CommonMark extension describing how hashtags should work, but I didn't find one.
